games-roguelike/angband/angband-4.1.3.ebuild

```diff
@@ -0,0 +1,113 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools eapi7-ver eutils gnome2-utils
+
+MAJOR_PV=$(ver_cut 1-2)
+
+DESCRIPTION="A roguelike dungeon exploration game based on the books of J.R.R. Tolkien"
+HOMEPAGE="https://rephial.org/"
+SRC_URI="https://rephial.org/downloads/${MAJOR_PV}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="ncurses sdl +sound X"
+
+RDEPEND="X? ( x11-libs/libX11 )
+	!ncurses? ( !X? ( !sdl? ( sys-libs/ncurses:0=[unicode] ) ) )
+	ncurses? ( sys-libs/ncurses:0=[unicode] )
+	sdl? ( media-libs/libsdl[video,X]
+		media-libs/sdl-image
+		media-libs/sdl-ttf
+		sound? ( media-libs/libsdl[sound] )
+			media-libs/sdl-mixer[mp3] )"
+DEPEND="${RDEPEND}
+	dev-python/docutils
+	virtual/pkgconfig"
+
+src_prepare() {
+	default
+
+	sed -i -e '/libpath/s#datarootdir#datadir#' configure.ac || die
+	sed -i -e "/^.SILENT/d" mk/buildsys.mk.in || die
+	sed -i -e '/^DOC =/s/=.*/=/' doc/Makefile || die
+
+	if use !sound ; then
+		sed -i -e 's/sounds//' lib/Makefile || die
+	fi
+
+	# Game constant files are now system config files in Angband, but
+	# users will be hidden from applying updates by default
+	{
+		echo "CONFIG_PROTECT_MASK=\"/etc/${PN}/customize/\""
+		echo "CONFIG_PROTECT_MASK=\"/etc/${PN}/gamedata/\""
+	} > "${T}"/99${PN} || die
+
+	eautoreconf
+}
+
+src_configure() {
+	local myconf
+
+	if use sdl; then
+		myconf="$(use_enable sound sdl-mixer)"
+	else
+		myconf="--disable-sdl-mixer"
+	fi
+
+	ac_cv_path_NCURSES_CONFIG="pkg-config ncursesw" \
+	econf \
+		--bindir="${EPREFIX}"/usr/bin \
+		--with-private-dirs \
+		$(use_enable X x11) \
+		$(use_enable sdl) \
+		$(use_enable ncurses curses) \
+		$(use !sdl && use !ncurses && use !X && \
+			echo --enable-curses) \
+		${myconf}
+}
+
+src_install() {
+	DOCS="changes.txt faq.txt readme.txt thanks.txt" \
+		default
+
+	dodoc doc/manual.html
+	doenvd "${T}"/99${PN}
```

Comment:
I am not 100 % sure about EAPI-6, but you should research if you can do the new env file without using `${T}`. 

```
newenvd - "99${PN}.conf" <<- _EOF_
	"CONFIG_PROTECT_MASK=\"/etc/${PN}/customize/\""
	"CONFIG_PROTECT_MASK=\"/etc/${PN}/gamedata/\""
_EOF_
```
(I don't remember wether that was introduced in EAPI-6 or -7)

Reply:
I couldn't find anything on this in devmanual. Why using `${T}` is bad?

Reply:
Other than introducing I/O stress for host system, adding extra lines to ebuild and dealing with it in multiple phases, when all could be done without those? "It's the old way".